role-make-version-changelog.sh

```diff
@@ -130,50 +135,86 @@ else
     fi
 fi
 if [ "$skip" = false ]; then
-    echo "If you want to continue, enter the new tag in the form ${allow_v}X.Y.Z"
-    echo "where X, Y, and Z are integers corresponding to the semantic"
-    echo "version based on the changes above."
-    echo "Or, just press Enter to skip this role and go to the next role."
+    ver_major=$(echo "${latest_tag}" | cut -d'.' -f 1)
```

Comment:
note that `latest_tag` can be empty at this point, or may contain something that doesn't look like a version (if someone mistakenly tagged a repo)
maybe something like this
```bash
if [[ "$latest_tag" =~ ^"$allow_v"([0-9]+)[.]([0-9]+)[.]([0-9]+)$ ]]; then
  ver_major="${BASH_REMATCH[1]}"
  ver_minor="${BASH_REMATCH[2]}"
  ver_patch="${BASH_REMATCH[3]}"
elif [ -z "$latest_tag" ]; then
  latest_tag=EMPTY
  ver_major=0
  ver_minor=0
  ver_patch=0
else
  echo ERROR: unexpected tag "$latest_tag"
  exit 1
fi
```
Also - if `ver_major` is 0, this means the role is still in prerelease - we should not bump `ver_major` automatically from 0 to 1 - I know the current code doesn't do automatic releases, but moving from `ver_major` 0 to 1 should be manual.

Reply:
Thanks a lot for your feedback, I applied everything, including not bumping major var if the version is a prerelease.
I've applied some formatting to long lines that you suggested, I assumed that you didn't suggest in multiline because of github comments limitations.
Please take a look.